index.html

```diff
@@ -2891,6 +2896,66 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
+        <h3>
+          <code title="">iarc_rating_id</code> member
```

Comment:
> Also if MS plans to show these in Edge at some point (say with a prompt) we need to find out what to do in that case. Talk to IARC?

We might need to define how to actually do that. This means pinging the IARC servers for validation, no? Is it just a GET request? Do we need any special HTTP headers? What happens if you get a 404? etc.